jag-ccd-application/src/main/java/ca/bc/gov/open/ccd/controllers/FileController.java

```diff
@@ -28,8 +27,11 @@
 @Endpoint
 public class FileController {
 
-    @Value("${ccd.host}")
-    private String host = "https://127.0.0.1/";
+    @Value("${ccd.host}" + "civil/")
```

Comment:
Is having 2 really necessary cant we just combine in the url building?

Reply:
changed and only one configured host now.  each endpoint will then have its special url host.